python/scripts/read_hfb_data.py

```diff
@@ -3,85 +3,135 @@
 import matplotlib.pyplot as plt
```

Comment:
Does this script really warrant being in the repo? It looks like your own personal test script. Typically these kind of things become irrelevant and then entropy takes hold and they break, and then 12 months later you're trying to figure out what it was all for anyway, and just remove it.

Reply:
I prefer keeping this in the repo as I like to use it to test the raw files written.

Reply:
Okay. I'm mostly of the opinion that if it's just your own personal test script, it doesn't belong in the main repo. Mostly out of tidyness, as if we all did that the repo would be full of random scripts. Maybe @andrerenard has an opinion on this one?